internal/pkg/service/common/etcdop/op/tracker.go

```diff
@@ -117,8 +120,8 @@ func (v *trackedTxn) Commit() (*etcd.TxnResponse, error) {
 	return r, err
 }
 
-func (v *TrackerKV) track(t TrackedOpType, key, rangeEnd []byte, count int64) {
-	record := TrackedOp{Type: t, Key: key, RangeEnd: rangeEnd, Count: count}
+func (v *TrackerKV) track(t TrackedOpType, key, rangeEnd []byte, count int64, kvs []*mvccpb.KeyValue) {
+	record := TrackedOp{Type: t, Key: key, RangeEnd: rangeEnd, Count: count, KVs: kvs}
```

Comment:
The slice is already in the memory, so the best approach is to store reference to the slice.

Slices are always passed by a reference:
- To be exact, slice header is copied (`firstElementAddr, len, cap`), but underlying array not.